modules/highgui/src/window_QT.cpp

```diff
@@ -1633,7 +1633,7 @@ CvWinProperties::~CvWinProperties()
 // CvWindow
 
 
-CvWindow::CvWindow(QString name, int arg2)
+CvWindow::CvWindow(const QString& name, int arg2)
```

Comment:
There is no corresponding change in the header.

Reply:
OK，I will check it again and build through locally